### PR TITLE
login-with-fb-gg

### PR DESCRIPTION
**Pull request by CaDuyDu**

### CODE REVIEW

**Comment by CaDuyDu** [comment]
ready

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def google_oauth2
+      @user = User.from_omniauth(request.env["omniauth.auth"])
```
Comment (huuphuc93) [inline app/controllers/users/omniauth_callbacks_controller.rb]: - indent, xóa()

File: app/models/user.rb [hunk 2]
```diff
@@ -24,6 +24,19 @@ def current_user? user
     self == user
   end
 
+  def self.from_omniauth(access_token)
+    data = access_token.info
+    user = User.where(:email => data["email"]).first
```
Comment (huuphuc93) [inline app/models/user.rb]: dùng find_by đi em

**Comment by CaDuyDu** [comment]
ready

**Pull request merged**